### PR TITLE
Add hostinfo plugin to report SSHd configuration

### PR DESCRIPTION
**Pull request by ryandub**
Parse output of `sshd -T` and gather SSH configuration data.

### CODE REVIEW

File: hostinfo/sshd.lua [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+local spawn = require('childprocess').spawn
+
+--[[ SSHd Variables ]]--
+local Info = HostInfo:extend()
+function Info:initialize()
+  HostInfo.initialize(self)
+end
+
+function Info:run(callback)
+  if os.type() ~= 'Linux' and os.type() ~= 'Darwin' then
```
Comment (pquerna) [inline hostinfo/sshd.lua]: isn't it really just, is `sshd` in $PATH, not an OS specific check?  eg, this should work on FreeBSD out of the box AFAIK.

Comment (ryandub) [inline hostinfo/sshd.lua]: Yes, as far as I know this is correct for most modern versions of `sshd`. Updated to remove this check. Thanks!

Should there be a check for `sshd` in the path or just return the system error through the error handler?

**Comment by rjemanuele** [comment]
This looks good to me.  +1 upon deciding to use something like execFile to simplify the code.  https://gist.github.com/rjemanuele/4ac7473464b74b9a721d

**Comment by ryandub** [comment]
Does #671 replace this? Should this be closed?

**Comment by rphillips** [comment]
favoring #671

**Pull request closed without merging**